### ISSUE TITLE
MVE loss to do list

### ISSUE THREAD

**Issue by beckynevin** [deepskies/DeepUQ#67]
- [x] Add error bars to error plots
- [x] Implement some metric (MSE?) to check if a model converges
- [x] Change plotting to validation set and training sets in grey
- [x] Annotate MSE on said plots alongside different types of loss
- [x] Double check variance versus std input to bnll loss
- [x] Sum or mean loss?
- [x] Run beta = 0 100 times, check convergence percentage
- [x] Run beta = 1 100 times, check convergence percentage
- [x] Run decreasing beta 100 times, check convergence percentage
- [x] NN architecture should match that of DER
